sql/sqltool/tool.go

```diff
@@ -351,6 +356,26 @@ func (d *FlywayDir) Files() ([]migrate.File, error) {
 	return ret, nil
 }
 
+// Checksum implements Dir.Checksum. By default, it calls Files() and creates a checksum from them.
+func (d *FlywayDir) Checksum() (migrate.HashFile, error) {
+	if d, ok := d.FS.(migrate.Dir); ok {
+		return d.Checksum()
+	}
+	files, err := d.Files()
+	if err != nil {
+		return nil, err
+	}
+	return migrate.NewHashFile(files)
+}
+
+// WriteFile implements Dir.WriteFile.
+func (d *FlywayDir) WriteFile(name string, b []byte) error {
+	if d, ok := d.FS.(migrate.Dir); ok {
+		return d.WriteFile(name, b)
+	}
+	return errors.New("sql/sqltool: cannot write to non-local directory")
```

Comment:
```suggestion
	return errors.New("sql/sqltool: write not supported")
```
This is more accurate, isn't it?